x/wasm/alias.go

```diff
@@ -1,4 +1,3 @@
-//nolint
```

Comment:
The linter and formatter where fighting on this line

Reply:
it would hate that line, great catch!